shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultResult.java

```diff
@@ -22,6 +22,15 @@
  */
 public final class FaultResult extends AbstractExtendable<FaultResult> {
 
+    public enum Status {
+        SUCCESS,
+        NO_SHORTCIRCUIT_DATA,
+        SOLVER_FAILURE,
+        FAILURE
+    }
+
+    private Status faultResultStatus;
```

Comment:
Let's make it `final`.

`status` is enough, faultResultStatus sounds redundant (and actually we could say it's more the  status of the "computation", than the status of the "result"). I see it's already done this way in the getter and serialization --> let's be consistent here too :)